src/mlpack/bindings/python/print_wrapper_py.cpp

```diff
@@ -64,7 +64,7 @@ void PrintWrapperPY(const std::string& category,
   }
 
   // Import different mlpack programs that are to be wrapped.
-  for(int i=0; i<methods.size(); i++)
+  for(long unsigned int i=0; i<methods.size(); i++)
```

Comment:
this should probably be `std::size_t` instead of `long unsigned int`

https://en.cppreference.com/w/cpp/types/size_t

The `long` type is not a reliable indicator of bit width:  https://en.cppreference.com/w/cpp/language/types